src/dropdown.rs

```diff
@@ -37,7 +37,7 @@ impl<T: Data> Dropdown<T> {
         self.window = Some(
             ctx.new_sub_window(
                 WindowConfig::default()
-                    .set_level(WindowLevel::DropDown)
+                    .set_level(WindowLevel::DropDown(ctx.window().clone()))
                     .set_position(origin)
```

Comment:
this also needs a change, right?

Reply:
RIght. But no point making this change because dropdowns arent handled in druid yet.